### ISSUE TITLE
Default green colour does not pass WCAG AA contrast checker

### ISSUE THREAD

**Issue by mudrd8mz** [Veloxnet-LMS/moodle-theme_vxg_boost#1]
Just a suggestion. When installing the theme, the default green colour `#78C2AD` does not meet the contrast check as per WCAG AA

https://webaim.org/resources/linkcontrastchecker/

For a lot of institutions, accessibility is a key factor when choosing a theme. It may help to promote the theme if the default behaviour was closer to accessibility standards.

**Comment by mudrd8mz** [comment]
It seems to affect some other places too, such as warning notifications.

![image](https://user-images.githubusercontent.com/85831/107497410-d356a400-6b92-11eb-8c09-c8e670b5108b.png)
